tasks/jenkins.rake

```diff
@@ -3,17 +3,24 @@ require 'English'
 begin
   require "ci/reporter/rake/test_unit"
   namespace :jenkins do
+    desc 'Sets up pre-CI environment for testing'
     task :unit => ["jenkins:setup:test_unit", 'rake:test']
 
     namespace :setup do
+      desc 'Sets up pre-CI environment for Minitest'
       task :pre_ci do
         ENV["CI_REPORTS"] = 'jenkins/reports/unit/'
```

Comment:
While you're at it, I think you can actually drop this. We now set it in CI: https://github.com/theforeman/jenkins-jobs/blob/a44d192c1549be46b3f0d85c254f4247c4ec9331/theforeman.org/pipelines/test/smart-proxy.groovy#L97